File: TD/TD2/Code/6.cpp
/* Calculer la somme des chiffres qui composent un nombre */
#include <iostream>

int main()
{
  int a;
  int b;
  
  b = 0;
  
  std::cout << "Donnes moi un chiffre : " << std::flush;
  std::cin >> a;
  
  while (a != 0) {
    b += a % 10;
    a /= 10;
  }
  
  std::cout << "La somme des chiffres est " << b << std::endl;
  
  return 0;
}

File: TD/TD2/Code/Python/6.py
#!/usr/bin/python
# Calculer la somme des chiffres qui composent un nombre
somme = 0

print('Donne moi une valeur : ', end='')
a = int(input())

while a != 0:
    somme += a % 10
    a = int(a / 10)

print('La somme des chiffres est égal à', somme)
